ebcl/tools/initrd/initrd.py

```diff
@@ -295,6 +315,19 @@ def create_initrd(self) -> Optional[str]:
         """ Create the initrd image.  """
         image_path = os.path.join(self.config.output_path, self.name)
 
+        # Create necessary directories
+        for dir_name in ['proc', 'sys', 'dev', 'sysroot', 'var', 'usr/bin',
+                         'tmp', 'run', 'root', 'usr', 'sbin', 'usr/lib', 'etc']:
+            self.config.fake.run_sudo(
+                f'mkdir -p {os.path.join(self.target_dir, dir_name)}')
+            self.config.fake.run_sudo(
+                f'chown 0:0 {os.path.join(self.target_dir, dir_name)}')
```

Comment:
While we are at it, this can be removed. If mkdir is called using sudo, it will always create the directory with "root:root", which is "0:0" or am I missing something?